src/components/Monitoring/index.js

```diff
@@ -1,37 +1,80 @@
 // @flow
 import * as React from 'react';
-import { Chart, Geom, Axis, Tooltip } from 'bizcharts';
-
-const sample_data = [
-  { stream: 'Cat Scan', size: 2400 },
-  { stream: "Driver's License", size: 655 },
-  { stream: 'Health Card', size: 980 },
-  { stream: 'X-Ray', size: 5320 },
-  { stream: 'Other', size: 3642 },
-];
-
-const cols = {
-  size: {
-    tickInterval: 1000,
-    alias: 'Size (KB)',
+import { Card, Col, Layout, Row } from 'antd';
+import { Graph } from 'react-d3-graph';
+import { observer } from 'mobx-react';
+import { observable } from 'mobx';
+
+import CurrentNode from 'components/Monitoring/CurrentNode';
+import _data from './data';
+
+const { Content } = Layout;
+
+const makeData = () => {
+  const data = {
+    nodes: [],
+    links: [],
+    focusedNodeId: 1,
+  };
+
+  _data.forEach(d => {
+    data.nodes.push(d);
+    data.links.push({ source: 1, target: d.id });
+  });
+
+  return data;
+};
+
+const myConfig = {
```

Comment:
This config is heavily extensible. I just didn’t have the time to improve on it.